models/_sector/nft/trades/solana/nft_solana_trades.sql

```diff
@@ -0,0 +1,83 @@
+{{
+    config(
+        schema = 'nft_solana'
+        , tags = ['dunesql']
+        , alias = alias('trades')
+        , materialized = 'view'
+        ,post_hook='{{ expose_spells(\'["solana"]\',
+                                    "sector",
+                                    "nft",
+                                    \'["ilemi"]\') }}'
+    )
+}}
+
+-- ,file_format = 'delta'
+-- ,incremental_strategy = 'merge'
+-- ,partition_by = ['project','block_month']
+-- ,unique_key = ['project','trade_category','outer_instruction_index','inner_instruction_index','account_metadata','tx_id']
+
+
+{% set solana_marketplaces = [
+    ref('magiceden_solana_trades')
+    , ref('tensorswap_solana_trades')
+] %}
+
+
+{% for marketplace in solana_marketplaces %}
+SELECT
+        blockchain,
+        project,
+        version,
+        cast(date_trunc('day', block_time) as date) as block_date,
+        cast(date_trunc('month', block_time) as date) as block_month,
+        block_time,
+        token_name,
+        token_symbol,
+        trade_type,
+        number_of_items,
+        trade_category,
+        buyer,
+        seller,
+        amount_raw,
+        amount_original,
+        amount_usd,
+        currency_symbol,
+        currency_address,
+        account_metadata,
+        account_master_edition,
+        account_mint,
+        verified_creator,
+        collection_mint,
+        project_program_id,
+        aggregator_name,
+        aggregator_address,
+        tx_id,
+        block_slot,
+        tx_signer,
+        taker_fee_amount_raw,
+        taker_fee_amount,
+        taker_fee_amount_usd,
+        taker_fee_percentage,
+        maker_fee_amount_raw,
+        maker_fee_amount,
+        maker_fee_amount_usd,
+        maker_fee_percentage,
+        royalty_fee_amount_raw,
+        royalty_fee_amount,
+        royalty_fee_amount_usd,
+        royalty_fee_percentage,
+        amm_fee_amount_raw,
+        amm_fee_amount,
+        amm_fee_amount_usd,
+        amm_fee_percentage,
+        instruction,
+        outer_instruction_index,
+        inner_instruction_index,
+        unique_trade_id,
+        row_number() over (partition by unique_trade_id order by tx_id) as duplicates_rank
```

Comment:
you built the `duplicates_rank` column, i'd imagine to match `nft.trades`, but don't filter down below where value is 1. can you add the filter too, or remove this column if not?

Reply:
ah okay. i'll remove